templates/otp.conf.j2

```diff
@@ -0,0 +1,9 @@
+description "start OpenTripPlanner process"
+
+start on {{ upstart_start_on }}
+stop on runlevel [!2345]
+
+chdir {{ otp_bin_dir }}
+script
+    exec sudo -u {{ otp_user }} ./otp -a --graphs {{ otp_data_dir }} --server
```

Comment:
Hahahaha yeah that'll do it.

Thanks for clarifying that a bit, I'll tweak this upstart script and add a (hopefully) sensible configurable default and add some of these notes to the README.

The philadelphia OSM data that this role defaults to seems to be happy loading in about 2Gb, as a point of reference.